### ISSUE TITLE
test data used during training

### ISSUE THREAD

**Issue by philippemiron** [donny-chan/pinn-torch#1]
Looking at the code in `data.py`:

```
idx = np.random.choice(train_data.shape[0], 1000, replace=False)
test_data = train_data[idx, :]
train_data = PinnDataset(train_data)
test_data = PinnDataset(test_data)
```

the testing data is used during training, this should be removed from the training dataset for appropriate testing of the model.

**Comment by chen-yingfa** [comment]
You are right, but this code is just an example for learning purposes, and the result should not be regarded as an evaluation of the methods. I will be updating this in a new commit though. Thanks.
